ops/ansible/ansible_upgrade.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+pip install ansible
```

Comment:
@thatarchguy, is there any reason you don't have the --upgrade flag on here, but you do in ansible_install.sh? Maybe we can ensure that a minimum version is installed (i.e. 2.0+)